app/__init__.py

```diff
@@ -8,5 +8,9 @@ def create_app():
     
     # Register the blueprints for endpoints in app instance
     app.register_blueprint(controllers.get_index_bp)
+    app.register_blueprint(controllers.get_Sign_Up)
+    
+    return app
```

Comment:
Please get rid of whitespaces and blanks lines here and elsewhere too. Please follow this for all the files and tag me once it's done, Thanks